garage/contrib/exp/core/policy.py

```diff
@@ -0,0 +1,33 @@
+import abc
+
+
+class Policy(abc.ABC):
```

Comment:
I'm not sure why this interface is necessary when `Agent` is nearly exactly the same.